src/App.scss

```diff
@@ -530,7 +530,13 @@ table {
     }
   }
 }
-
+.stickyTable {
+  thead {
+    position: sticky;
+    top: 0;
+    background-color: white;
```

Comment:
Let's not change the background color :)

Reply:
actually it makes the header translucent, and hinders visibility, 
check this out 👇 
![Screenshot 2020-03-26 at 12 10 42 PM](https://user-images.githubusercontent.com/10740210/77617775-f5cfd680-6f5a-11ea-9b89-146c126e5f1d.png)